IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Types/IfcArchElementTypeEnum/Documentation.md

```diff
@@ -0,0 +1 @@
+This enumeration defines the different types of elements an _IfcArchElement_ or _IfcArchElementType_ object can fulfill.
```

Comment:
Feel free to reject. Just thought it sounds better and fits better with the relationship.
```suggestion
This enumeration defines the different types of elements an _IfcArchElement_ or _IfcArchElementType_ object can be defined as.
```